apps/admin/src/containers/FirestoreSignin/firestore.ts

```diff
@@ -16,7 +16,7 @@ export async function signIn() {
     if (!username || !password)
         return false;
 
-    return await init({username, password});
+    return init({username, password});
```

Comment:
No await necessary?

Reply:
nope, turns out it was necessary, I've put it back in in the next PR (d'oh)